README.md: how to run unit test (#403)
aiidalab/aiidalab-qe: README.md

```
The pre-commit checks should now be automatically executed prior to each commit.

> To run unit tests for app development, you need to run `pytest` from `aiida-core-services` conda environment, e.g. with:

Reviewer: ```suggestion
To run unit tests in the AiiDAlab container, you need to run `pytest` from within the `aiida-core-services` conda environment:
```
